test/clientjs/test_base.js

```diff
@@ -220,15 +220,73 @@ describe('initializeToggles', function() {
 
 describe('initializeIframes', function() {
   it('triggers callbacks on window resize', function() {
-    const srcUrl = '';
-    const iframe = $(
-      `<iframe src="${srcUrl}" height="20" width="20"></iframe>`
-    );
+    sinon.stub($.fn, 'outerHeight').returns(30);
+    const iframe = $('<iframe height="20">');
     iframe.appendTo('body');
     base.initializeIframes();
+
+    expect(iframe.get(0).height).to.equal('30');
+    expect($.fn.outerHeight).to.have.been.calledOnceWithExactly(true);
+
     iframe.trigger('load');
+
+    expect($.fn.outerHeight).to.have.been.calledTwice;
+
     $(window).trigger('resize');
-    // @@@ This is not a very useful or informative assertion:
-    expect(iframe.get(0).height).to.equal('20');
+
+    expect($.fn.outerHeight).to.have.been.calledThrice;
+
+    iframe.remove();
+    $.fn.outerHeight.restore();
```

Comment:
One thing I noticed (and you'll see I had to write tests this way) is that if the assertion fails, and stub-restoration you do at the end won't be run, potentially causing cascading failures. I'd suggest putting this either in an `afterEach` (probably overkill) or wrapping the test body in a `try` and putting this in a `finally`?

Reply:
Ah, that's a great suggestion. I saw you'd done the `try/finally` bit, but I didn't register the reason for it. 👍